src/xenia/ui/vulkan/vulkan_util.cc

```diff
@@ -28,7 +28,7 @@ Version Version::Parse(uint32_t value) {
   Version version;
   version.major = VK_VERSION_MAJOR(value);
   version.minor = VK_VERSION_MINOR(value);
-  version.patch = VK_VERSION_PATCH(value);
+  version.patch = VK_VERSION_PATCH(VK_HEADER_VERSION);
```

Comment:
This doesn't look quite right - this should still be parsing the input value.

Reply:
I don't remember why I did this but didn't mean to commit it. Thanks for the catch.